contracts/abstracts/BlockRewardBase.sol

```diff
@@ -6,53 +6,79 @@ import "../eternal-storage/OwnedEternalStorage.sol";
 import "../libs/SafeMath.sol";
 
 
+/// @title The Block Reward Base Contract
+/// @author POA Networks, Inc.
+/// @notice The block reward contract is responsible for rewarding validators and those who have delegated stake to
+/// them.  It therefore provides the basis for validators' and delegators' profitability.  Additionally, it is
+/// responsible for determining the amount that validators must reward themselves and their delegators.  It therefore
+/// controls the supply of native tokens on the blockchain.
+///
+/// This contract is an abstract base that contains functions used both by AuRa and HBBFT consensus.
 contract BlockRewardBase is OwnedEternalStorage, IBlockReward {
     using SafeMath for uint256;
 
-    // ============================================== Constants =======================================================
-
-    // This address must be set before deploy
-    address public constant VALIDATOR_SET_CONTRACT = address(0x1000000000000000000000000000000000000001);
-
     // ================================================ Events ========================================================
 
+    /// @dev This event fires when an amount is awarded to a designated receiver of funds.
+    /// @param amount The amount of funds transferred, in wei.
+    /// @param receiver The receiver of the funds.
+    /// @param bridge The bridge used to transfer the funds, or 0x0 if no bridge was used.
```

Comment:
Are funds ever sent directly via the bridge from the block reward contract?

In general, this contract mentions the bridge a lot, and I don't understand why the block reward contract needs to know about the bridge.

Reply:
The bridge uses BlockReward's `addExtraReceiver` function to mint some native coins when needed.

Reply:
I see; but in our case the bridge's local end will be ERC20, not native, right?
(I'm fine with keeping that functionality in the contract, I'm just wondering whether we'll use it.)

Reply:
We will have two bridges connected with dPoS network, one of them will use minting native coins by BlockReward (like it happens in xDai network at the moment).